src/integrationTest/groovy/wooga/gradle/utils/PropertyQueryTaskWriter.groovy

```diff
@@ -65,8 +65,8 @@ class PropertyQueryTaskWriter extends BasePropertyQueryTaskWriter {
     /**
      * @return True if the property's toString() matches the given value
      */
-    Boolean matches(ExecutionResult result, Object value) {
-        result.standardOutput.contains("${path}${separator}${value}")
+    Boolean matches(String stdOut, Object value) {
+        stdOut.contains("${path}${separator}${value}")
```

Comment:
This feels weird. I see the reason for the old method to have a quick way of checking if the ExecutionResult contains something. Which I would suggest to check for both stdout and stderr because gradle switched in the past where certain messages got posted to. But changing this to String makes not sense. Can't you add an overload and call this one from the original method. Also keep in mind that most of this is now part of the gradle-commons-test lib. And it would be cool if we can simply switch to that one.

Reply:
I'll add the overload. And I agree that would be nice if we switch this to the gradle-commons-test. I have some additions to do there as well that can be applied to this project,